docs/user/development.rst

```diff
@@ -6,30 +6,32 @@ GitHub. The development began in September 2014.
 
 It is developed on GitHub: https://github.com/Lasagne/Lasagne
```

Comment:
I'd suggest to merge this with the initial paragraph so that GitHub is not mentioned twice. Actually, neither the authors nor the project start time are really relevant in a document on how to contribute to the project, but it could go like:
```
The ``Lasagne`` project was started by Sander Dieleman in September 2014. It is developed by
a [core team of seven people](https://github.com/orgs/Lasagne/teams/core-team) and
[numerous additional contributors](https://github.com/Lasagne/Lasagne/graphs/contributors)
on GitHub: https://github.com/Lasagne/Lasagne
```